{{cookiecutter.project_name|replace(" ", "")}}/.github/etc/dictionary.txt

```diff
@@ -1,7 +1,9 @@
 {%- set fragments = cookiecutter.project_name.split('_')
+                  + cookiecutter.project_name.split('-')
```

Comment:
Note that this is already de-duplicated and sorted by the post generation hook, so this is fine